### ISSUE TITLE
File modification date not updated

### ISSUE THREAD

**Issue by jomey** [cryptomator/cryptomator-ios#127]
## Basic Info
Version: 1.5.0 (718)

## Description
After uploading and replacing a document with a new version, the modification date does not get updated until you re-open the vault.

## Steps
1. Open a vault
2. Select a document to edit and save that to the ‘Files’ app
    * Keep the view of the vault open
3. Switch to ‘Files’ app
4. Edit the document, save, and upload to ‘Cryptomator’
5. Go back to ‘Cryptomator’ with the open vault
-> The file shows the old modification time, although it was just replaced
6. Go back to the vault overview
7. Re-open the same vault
-> The file now shows the correct date

**Comment by jomey** [comment]
@tobihagemann - Is this not the proper way of reporting bugs for the iOS app? I took the instructions from within the app and there was no response so far.

**Comment by tobihagemann** [comment]
It absolutely is. We are kind of in a weird position with the iOS app right now because we are working on a huge update, which would make most bug reports obsolete. However, we're still many months away from a release.

Just to be sure: Have you tried the "pull-to-refresh" gesture in step 5? This should also "update" the contents of the current directory.

**Comment by jomey** [comment]
Thanks for the update. Maybe you could put an announcement/heads-up in the bug reporting instructions to let people know about this. Might be more people wondering, why there is little activity.

I am aware of the the update and it "solves" the stale issue. However, I would not expect having to do this and the app would refresh, once there is a change to a file detected.

**Comment by tobihagemann** [comment]
You're absolutely right. We have been thinking about an announcement for quite some time because we're very excited about the "huge" update but that's something for another time.

Thinking about this issue, I think it's something that we actually have to re-test with the new app. I don't think that it makes too much sense to fix it now because it's not that trivial to synchronize the state between the main app and an app extension.
